lib/server.js

```diff
@@ -382,7 +382,9 @@ Server.prototype.start = function start(cb) {
 
   function emitListeningSignal(serialCb) {
     debug('emitting listening event');
-    self.emit('listening', self._httpServer.address());
+    var addr = self._httpServer.address();
+    self.emit('listening', addr);
+    self._meshApp.emit('started', self.port);
```

Comment:
Ah, the reason this doesn't break anything is because the value isn't actually used by anything..